### PR TITLE
[1LP][RFR]Fixing navigation on host collection via Compute-Infrastructure-Providers

### PR DESCRIPTION
**Pull request by kedark3**
Purpose or Intent
=================

__Fixing__ host collection navigation to use correct view when navigating via Compute->Infrastructure->Providers.

{{pytest: cfme/tests/v2v/test_v2v_migrations.py --use-provider rhv42 --use-provider vsphere65-nested --provider-limit 2 -vvvv --long-running  -k 'test_single_datastore_single_vm_migration'}}

### CODE REVIEW

File: cfme/common/host_views.py [hunk 1]
```diff
@@ -276,5 +276,6 @@ class ProviderAllHostsView(HostsView):
     def is_displayed(self):
         return (
             self.navigation.currently_selected == ["Compute", "Infrastructure", "Providers"] and
-            self.title.text == "{} (All Managed Hosts)".format(self.context["object"].name)
+            self.title.text == "{} (All Managed Hosts)".format(self.context["object"]
```
Comment (kedark3) [inline cfme/common/host_views.py]: This was required as HostCollection object did not have `name` attribute in itself

Comment (mshriver) [inline cfme/common/host_views.py]: consider breaking this after `.format(`

```
thing == '{} (title part)'.format(
    self.some_attributes.name
)

**Comment by kedark3** [comment]
@mshriver if this does not timeout/itemnotfound on host page in PRT then we are good to merge it.

**Comment by kedark3** [comment]
Linking to #7960

File: cfme/infrastructure/host.py [hunk 2]
```diff
@@ -634,7 +635,14 @@ def create_from_config(self, host_key):
 
 @navigator.register(HostsCollection)
 class All(CFMENavigateStep):
-    VIEW = HostsView
+
+    @property
+    def VIEW(self):
```
Comment (digitronik) [inline cfme/infrastructure/host.py]: please use  `# noqa` its not as per definition standard.

Comment (kedark3) [reply cfme/infrastructure/host.py]: will do. But, what is that? sorry I don't know.

File: cfme/v2v/migrations.py [hunk 3]
```diff
@@ -414,7 +414,8 @@ class vms(View):  # noqa
 
         @property
         def is_displayed(self):
-            return self.filter_by_dropdown.is_displayed
+            return (self.filter_by_dropdown.is_displayed and
+                 (len(self.browser.elements(".//div[contains(@class,'spinner')]")) == 0))
```
Comment (mshriver) [inline cfme/v2v/migrations.py]: Where are you seeing this behavior?

ensure_page_safe should be handling this before navigation is attempted.

Comment (kedark3) [reply cfme/v2v/migrations.py]: it happens in modal.

**Pull request merged**